src/components/display/Dropdown.jsx

```diff
@@ -308,10 +324,19 @@ const Dropdown = React.forwardRef(function DropdownFn(
 
 	const clickOutsidePopper = useCallback(
 		(e) => {
-			dropdownRef.current &&
+			if (
+				dropdownRef.current &&
 				e.target !== dropdownRef.current &&
 				!dropdownRef.current.contains(e.target) &&
+				// check if the attribute is in the event path
+				!reduce(
+					e.path,
```

Comment:
e.path in undefined in Firefox, it is a non-standard attribute. This makes the keepOpen not working

Reply:
Didn't notice that, thanks